### PR TITLE
Added activeadmin

### PR DESCRIPTION
**Pull request by posixpascal**

### CODE REVIEW

**Comment by tmandke** [comment]
@posixpascal is this done or is there more to come??

**Comment by posixpascal** [comment]
We can already merge this, but basically it's just the dashboard with no real data. Also I added a few models to activeadmin which miss a `permit_params` function call. 

Keep this open for another 2 days so I can finish the rest of activeadmin.

File: app/models/user/identity.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ class User::Identity < OmniAuth::Identity::Models::ActiveRecord
     self.user ||= User.new
   end
 
-  delegate :first_name, :last_name, :location, :primary_circle_id, to: :user
+  delegate :first_name, :last_name, :location, :primary_circle_id, :admin?, to: :user
```
Comment (phil-monroe) [inline app/models/user/identity.rb]: Do we need access to the admin flag on the identity model?

**Comment by phil-monroe** [comment]
Looks like a good start on active admin. thanks @posixpascal!

**Comment by posixpascal** [comment]
We can merge this so other people can take a look at activeadmin. I'll fix the `permit_params` etc. later.

**Pull request merged**